config/configload/merge.go

```diff
@@ -592,6 +621,34 @@ func absInBackends(block *hclsyntax.Block) error {
 	return nil
 }
 
+func addProxy(block *hclsyntax.Block, proxies map[string]*hclsyntax.Block) error {
+	if attr, ok := block.Body.Attributes[proxy]; ok {
+		reference, err := attrStringValue(attr)
+		if err != nil {
+			return err
+		}
+
+		delete(block.Body.Attributes, proxy)
+
+		if proxyBlock, ok := proxies[reference]; !ok {
+			return newMergeError(errMissingReferencedProxy, block)
```

Comment:
* This should create an error pointing to the *attribute* (attribute's `Range`), not to the *endpoint block* (endpoint block's `DefRange`).
* Test this error

Reply:
Done

Reply:
* Test this error

is still TBD